### PR TITLE
Adds lawnmower and gatfruit to the syndie botanist's uplink

### PR DESCRIPTION
**Pull request by karmaisblackandbluepilled**
https://user-images.githubusercontent.com/54416597/126915943-289d4f2c-8a83-43c2-8c47-90fe08cb92ed.mp4

you already know how the gatfruit looks
also the lawnmower doesn't do this unless emagged

### CODE REVIEW

**Comment by karmaisblackandbluepilled** [comment]
oh yeah literally none of these prs are actually coded by me, they're ports from hippie, not like any of the 2 people still there will care

File: code/modules/vehicles/pimpin_ride.dm [hunk 1]
```diff
@@ -74,3 +74,65 @@
...
+	to_chat(user, "<span class='warning'>You disable the safety mechanisms on \the [src].</span>")
+	emagged = TRUE
+
+/obj/vehicle/ridden/lawnmower/Bump(atom/A)
+	if(emagged)
+		if(isliving(A))
+			var/mob/living/M = A
+			M.adjustBruteLoss(25)
+			var/atom/newLoc = get_edge_target_turf(M, get_dir(src, get_step_away(M, src)))
+			M.throw_at(newLoc, 4, 1)
```
Comment (Djiq) [inline code/modules/vehicles/pimpin_ride.dm]:
```suggestion
	if(!emagged || !isliving(A))
		return
	var/mob/living/M = A
	M.adjustBruteLoss(25)
	var/atom/newLoc = get_edge_target_turf(M, get_dir(src, get_step_away(M, src)))
	M.throw_at(newLoc, 4, 1)
```

Comment (Djiq) [reply code/modules/vehicles/pimpin_ride.dm]: a small rearranging just since it will look better

File: code/modules/vehicles/pimpin_ride.dm [hunk 2]
```diff
@@ -74,3 +74,65 @@
...
+
+/obj/vehicle/ridden/lawnmower
+	name = "lawn mower"
+	desc = "Equipped with reliable safeties to prevent <i>accidents</i> in the workplace."
+	icon = 'icons/obj/vehicles.dmi'
+	icon_state = "lawnmower"
+	var/emagged = FALSE
+	var/list/drive_sounds = list('sound/effects/mowermove1.ogg', 'sound/effects/mowermove2.ogg')
+	var/list/gib_sounds = list('sound/effects/mowermovesquish.ogg')
+	var/driver
```
Comment (TheChosenEvilOne) [inline code/modules/vehicles/pimpin_ride.dm]: unused var?

**Review by TheChosenEvilOne — APPROVED** [review]
LGTM

File: code/modules/vehicles/pimpin_ride.dm (hunk 2 above)
Comment (karmaisblackandbluepilled) [reply code/modules/vehicles/pimpin_ride.dm:86]: probably

**Comment by EdgeLordExe** [comment]
looks good to me

**Pull request merged**